composer.json

```diff
@@ -16,6 +16,7 @@
     "aws/aws-sdk-php": "^3.183.7",
     "dvsa/contracts": "^1.1",
     "firebase/php-jwt": "^5.2",
+    "guzzlehttp/guzzle": "^7.3",
```

Comment:
Could we set as `^6.0 || ^7.0` if possible - MOT is in a transition stage to PHP 8.0 and we still use Guzzle 6 in various places.

Reply:
Also as would be a new release give it a version :-)